### DISCUSSION TITLE
Surface metrics in version 2

### DISCUSSION THREAD

**Discussion by palomabernabe** [MIC-DKFZ/nnUNet#1538] [Q&A]
Hello @FabianIsensee,

I would like to know if there is some updated code to calculate the Hausdorff Distance, the Hausdorff Distance 95 and the Average Surface Distance.
I used to use the code available here: [nnUNet/nnunet/evalaution/metric.py](https://github.com/MIC-DKFZ/nnUNet/blob/nnunetv1/nnunet/evaluation/metrics.py), but I can't find an updated version in v2. If there's no update for this, is the code in v1 still good or are there any issues with it?

Thank you!

**Comment by NouraAAR** [comment]
Hello,

Did you eventually find an answer/solution? 
I also would like to calculate different metrics from the file [nnUNet/nnunet/evalaution/metric.py](https://github.com/MIC-DKFZ/nnUNet/blob/nnunetv1/nnunet/evaluation/metrics.py) but I am not sure how/where to use it. Did you eventually figure it out?

Thank you

**Comment by palomabernabe** [comment]
Hi!

I didn’t get any answer. I think I created my own code, adapted to the
characteristics of my project and my data, to calculate the metrics that I
needed. Let me know if there is anything else that I might be able to help
with.

Kind regards,

Paloma
